src/main/java/frc/robot/Input.java

```diff
@@ -16,7 +16,7 @@ private Input() {
 
   private static final Joystick mRotJoystick = new Joystick(JoystickConstants.ROT_JOYSTICK_PORT);
   private static final Joystick mTransJoystick = new Joystick(JoystickConstants.TRANS_JOY_PORT);
-  private static final XboxController mXboxController = new XboxController(JoystickConstants.XBOX_CONTROLLER_PORT);
+  public static final XboxController mXBox = new XboxController(JoystickConstants.XBOX_PORT);
```

Comment:
This isn't great, is there a way you can avoid exposing a field publically?

Reply:
done

Reply:
Er, how is it done? The field is still public.

Reply:
Should be fixed now